src/components/campaignLandingPage/template.tsx

```diff
@@ -0,0 +1,1212 @@
+type CampaignLandingPageProps = {
+  title: string
+}
+
+export function CampaignLandingPage({ title }: CampaignLandingPageProps) {
+  // TODO: Resolve these placeholders
+  const placeholders = {
+    // Hero section
+    fieldHeroBlurb: 'This is a blurb for the hero element',
+    fieldPrimaryCallToAction: {
+      entity: {
+        fieldButtonLink: { url: { path: '#' } },
+        fieldButtonLabel: 'Primary Action',
+      },
+    },
```

Comment:
This is just a placeholder, so it's fine for this scaffolding, but I _did_ want to call this out for posterity: We won't be nesting properties inside `.entity` objects. Content build does that as an artifact of the GraphQL query, but the JSON:API won't nest them like that, so there's no reason for us to add an additional layer to our formatted types.